src/components/section/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, ReactNode } from 'react';
+import {
+  Box,
+  BoxProps,
+  chakra,
+  RecipeVariantProps,
+  useSlotRecipe,
+} from '@chakra-ui/react';
+
+import { sectionRecipe } from 'src/themes/shared/slotRecipes/section';
+
+import { Stack } from '../stack';
+
+type SectionVariantProps = RecipeVariantProps<typeof sectionRecipe>;
+
+export type SectionProps = SectionVariantProps & {
+  variant?: 'hero' | 'regular';
+  title: string;
+  text?: string;
+  image?: ReactNode;
+  maxImgW?: BoxProps['maxWidth'];
+};
+
+export const Section: FC<SectionProps> = ({ maxImgW = '2xl', ...props }) => {
+  // const recipe = useSlotRecipe({ key: 'section' });
+  const recipe = useSlotRecipe({ recipe: sectionRecipe });
+  const [recipeProps, componentProps] = recipe.splitVariantProps(props);
+  const styles = recipe(recipeProps);
+
+  const { title, text, image } = componentProps;
+
+  return (
+    <Stack direction={{ '2xs': 'column', md: 'row' }} gap="xl">
+      {image ? (
+        <Box maxW={maxImgW} flexShrink={0}>
+          {image}
+        </Box>
+      ) : null}
+      <Stack gap="md">
+        <chakra.span css={styles.title}>{title}</chakra.span>
+        {text ? <chakra.span css={styles.text}>{text}</chakra.span> : null}
+      </Stack>
+    </Stack>
+  );
+};
+
+Section.displayName = 'Section';
```

Comment:
I think you don't need an explicit display name if you have a named component created in the package. We override it for re-exported `forwardRef`s